inst/eDISH_app/modules/renderSettings/renderSettings.R

```diff
@@ -172,19 +181,19 @@ renderSettings <- function(input, output, session, data, settings, status){
                                       values = input$`analysisFlag--values`)
       }
     }
-    
-    if (!is.null(input$filters)){
-      for (i in 1:length(input$filters)){
-        settings$filters[[i]] <- list(value_col = input$filters[[i]],
-                                      label = input$filters[[i]])
-      }
-    }
-    if (!is.null(input$group_cols)){
-      for (i in 1:length(input$group_cols)){
-        settings$group_cols[[i]] <- list(value_col = input$group_cols[[i]],
-                                         label = input$group_cols[[i]])
-      }
-    }
+    # 
+    # if (!is.null(input$filters)){
+    #   for (i in 1:length(input$filters)){
+    #     settings$filters[[i]] <- list(value_col = input$filters[[i]],
+    #                                   label = input$filters[[i]])
+    #   }
+    # }
+    # if (!is.null(input$group_cols)){
+    #   for (i in 1:length(input$group_cols)){
+    #     settings$group_cols[[i]] <- list(value_col = input$group_cols[[i]],
+    #                                      label = input$group_cols[[i]])
+    #   }
+    # }
```

Comment:
Not needed anymore? Or pasted elsewhere?

Reply:
Not needed since filters and group_cols can be specified as a list of 1+ values, or a vector of 2+ values (no label needed).  Will remove!